### PR TITLE
add Message and ChatRoom tables

### PR DESCRIPTION
**Pull request by HashEngineering**
Questions are in the code.

### CODE REVIEW

File: database/init.sql [hunk 1]
```diff
@@ -10,4 +10,20 @@ CREATE TABLE Users (
     PRIMARY KEY (id)
 );
 
+CREATE TABLE Message (
+    id CHAR(36),
+    text VARCHAR(1024),
+    room_id INT,
+    timestamp INT,
+    user_id CHAR(36),
```
Comment (HashEngineering) [inline database/init.sql:18]: Should these be foreign keys (room_id and user_id)?

File: database/init.sql [hunk 2]
```diff
@@ -10,4 +10,20 @@ CREATE TABLE Users (
     PRIMARY KEY (id)
 );
 
+CREATE TABLE Message (
+    id CHAR(36),
```
Comment (HashEngineering) [inline database/init.sql:14]: Not sure what type the message id should be.

File: database/init.sql [hunk 3]
```diff
@@ -10,4 +10,20 @@ CREATE TABLE Users (
...
+    text VARCHAR(1024),
+    room_id INT,
+    timestamp INT,
+    user_id CHAR(36),
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE ChatRoom (
+    id CHAR(36),
+    users CHAR(360),
```
Comment (HashEngineering) [inline database/init.sql:24]: Is this intended to be a list of users in the room?

**Comment by AaronLieb** [comment]
Going to merge because we do not rely on the new database information yet. Please create the new table for user to room relationships and add it in another pull request.

**Pull request merged**